### DISCUSSION TITLE
Season 2024-2025 update

### DISCUSSION THREAD

**Discussion by Cusco69** [kyleskom/NBA-Machine-Learning-Sports-Betting#445] [Ideas]
Hi,
Anyone know to make the update to the new season 2024-2025? I think is not automatic.... what files need to change to make the update?

**Comment by Tanstones630** [comment]
cd src/Process-Data
python -m Get_Data
python -m Get_Odds_Data
python -m Create_Games

This is a full data update. if you only want to update 24-25 data,I suggest you take a look at the code.and edit it.
Get_Data.py
Get_Odds_Data.py
Create_Games.py
config.toml

**Reply by etonyetang** [reply]
You can just adding a checking on the Get_Data if season existing in the datasets than don't fetch it otherwise fetching the season 24-25...

**Comment by yyuankm1976** [comment]
Do we also need to run Fix_Odds_Date_Format.py?
